dolphin/processor/files.py

```diff
@@ -290,7 +290,7 @@ def save_output_h5(self, lens_name, model_id, output):
                                             )
                     subgroup.create_dataset('param_list',
                                             data=np.array(single_output[2],
-                                                          dtype='S10')
+                                                          dtype='S25')
```

Comment:
Does choosing "S25" solve the clipped parameter name issue? I thought, it was lenstronomy that was doing the clipping to begin with.

Reply:
Yes, surprisingly it does. I also changed the parameter name font from 10 to 8 to accommodate the longer parameter name